### ISSUE TITLE
[PRE REVIEW]: studentlife: Tidy Handling and Navigation of a Valuable Mobile-Health Dataset

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#1551]
**Submitting author:** @frycast (<a href="http://orcid.org/0000-0001-6032-0522">Daniel Fryer</a>)
**Repository:** <a href="github.com/frycast/studentlife" target ="_blank">https://github.com/frycast/studentlife</a>
**Version:** v1.0.0
**Editor:** @arokem
**Reviewers:** @klmedeiros

**Author instructions**

Thanks for submitting your paper to JOSS @frycast. **Currently, there isn't an JOSS editor assigned** to your paper.

@frycast if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@whedon commands
```

**What happens now?**

This submission is currently in a `pre-review` state which means we are waiting for an editor to be assigned and for them to find some reviewers for your submission. This may take anything between a few hours to a couple of weeks. Thanks for your patience :smile_cat:

You can help the editor by looking at [this list of potential reviewers](https://bit.ly/joss-reviewers) to identify individuals who might be able to review your submission (please start at the bottom of the list). Also, feel free to suggest individuals who are not on this list by mentioning their GitHub handles here.

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #1551 with the following error: 

 Can't find any papers to compile :-(

**Comment by labarba** [comment]
👋  @frycast — the GitHub repo URL you provided with the submission is giving a 404. Can you post here the correct URL?

**Comment by labarba** [comment]
@arokem — the author requested you as handling editor, but I see that your load is on the high side right now. Are you willing to take another one? I can look for someone else if all your papers are mid-flight. Let me know!

**Comment by labarba** [comment]
I think this is the correct URL — pls. confirm @ frycast
https://github.com/frycast/studentlife

**Comment by arokem** [comment]
I can take this on.

**Comment by arokem** [comment]
@whedon assign arokem as editor

**Comment by whedon** [comment]
OK, the editor is arokem

**Comment by arokem** [comment]
@frycast, @labarba : I've edited the message at the top of this page to add the `https://` to the link, so that it resolves properly.

**Comment by arokem** [comment]
@frycast, as mentioned by whedon [above ](https://github.com/openjournals/joss-reviews/issues/1551#issuecomment-509014714), the paper, currently under a `JOSS-paper` folder can't be automatically found in this location. Could you please move it to a folder called `paper`?

**Comment by arokem** [comment]
:wave: @tgerke : would you be willing to review this paper for JOSS? Your expertise in public health and R would be helpful here. 

:wave: @pacoramon : would be willing to review this paper for JOSS? Your expertise in health and medicine, as well as statistical software would be helpful here. 

:wave: @borishejblum : would you be willing to review this paper for JOSS? Your expertise in epidemiology and R would be helpful here.

**Comment by labarba** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #1551 with the following error: 

 Can't find any papers to compile :-(

**Comment by labarba** [comment]
I thought it couldn't be found because the GitHub URL provided was wrong, and once you fixed that URL `whedon` would find it?

**Comment by frycast** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #1551 with the following error: 

 Can't find any papers to compile :-(

**Comment by frycast** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #1551 with the following error: 

 Can't find any papers to compile :-(

**Comment by frycast** [comment]
I tried to generate pdf with paper files in folder 'paper' and then I tried to generate pdf with them in the package root. I've now moved them back into folder 'paper'.

**Comment by frycast** [comment]
> 
> 
> I think this is the correct URL — pls. confirm @ frycast
> https://github.com/frycast/studentlife

Yes this is correct

**Comment by arokem** [comment]
Looks like it's summer. Let's see if we can find someone to respond to a request to review this.

**Comment by arokem** [comment]
:wave: @corinne-riddell : would be willing to review this paper for JOSS? Your expertise in epidemiology and R would be helpful here.

:wave: @jenniferthompson : would be willing to review this paper for JOSS? Your expertise in public health and R would be helpful here.

:wave: @Jminnier : would be willing to review this paper for JOSS? Your expertise in biostatistics and R would be helpful here

**Comment by jminnier** [comment]
Hi @arokem, sure!

**Comment by arokem** [comment]
Fantastic! Thanks for the quick reply. I'll assign you as a reviewer and wait for at least one more person to sign up.

**Comment by arokem** [comment]
@whedon assign @jminnier as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @jminnier

**Comment by arokem** [comment]
👋 @Rmadillo : would be willing to review this paper for JOSS? Your expertise in health informatics and R would be helpful here.

👋 @klmedeiros : would you be willing to review this paper for JOSS? Your expertise in public health, epi and R would be helpful here. 

👋 @acolum : would be willing to review this paper for JOSS? Your expertise in public health, epidemiology and R would be helpful here.

**Comment by Rmadillo** [comment]
Hi, I am on vacation until July 29, so I would be happy to do so after that date if you still need it.

On Jul 17, 2019, at 4:29 AM, Ariel Rokem <notifications@github.com<mailto:notifications@github.com>> wrote:


👋 @Rmadillo<https://github.com/Rmadillo> : would be willing to review this paper for JOSS? Your expertise in health informatics and R would be helpful here.

👋 @klmedeiros<https://github.com/klmedeiros> : would you be willing to review this paper for JOSS? Your expertise in public health, epi and R would be helpful here.

👋 @acolum<https://github.com/acolum> : would be willing to review this paper for JOSS? Your expertise in public health, epidemiology and R would be helpful here.

**Comment by klmedeiros** [comment]
@arokem I would be happy to do so. Is there a template or place where the review particularly needs to be submitted?

**Comment by arokem** [comment]
@whedon assign @klmedeiros as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @klmedeiros

**Comment by arokem** [comment]
@klmedeiros : thanks for taking this on (@Rmadillo : thanks! We'll call on you some other time -- enjoy your vacation!)

**Comment by arokem** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/1587. Feel free to close this issue now!

**Comment by arokem** [comment]
@jminnier, @klmedeiros : the review will take place in #1587. Please head over there for a checklist of the items that we would like you to review and to post your comments. See also https://joss.readthedocs.io/en/latest/reviewer_guidelines.html for guidelines.

**Comment by arokem** [comment]
Whoops. I believe that this one should have been closed when I started #1587. Closing now.
